src/App.tsx

```diff
@@ -14,11 +14,70 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string,
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1 className="App__title">{this.state.selectedGood.length > 1 ? `${this.state.selectedGood} is selected` : 'No goods selected'}</h1>
```

Comment:
```suggestion
        <h1 className="App__title">
          {this.state.selectedGood
            ? `${this.state.selectedGood} is selected`
            : 'No goods selected'
          }
        </h1>
```

Reply:
Just make your code clear, and will be better to use destructuring at the start of the render method for getting main values from state